src/routes/types.routes.ts

```diff
@@ -18,10 +18,10 @@ export type RootStackParamList = {
 	Accounts: undefined;
 	BackupSettings: undefined;
 	Transactions: undefined;
-	Save: undefined;
-	Deposit: undefined;
+	SaveScreen: undefined;
+	DepositScreen: undefined;
 	USDCoin: undefined;
-	DepositSuccess: undefined;
+	DepositSuccessScreen: undefined;
```

Comment:
looks like we don't use the word "Screen" after the screen name here. Can we follow this pattern or change the others to include "Screen"?